go.mod

```diff
@@ -65,3 +70,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.8.3
 	sigs.k8s.io/structured-merge-diff/v4 v4.1.0 // indirect
 )
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v0.18.33-0.20210409164018-2cf6f0aca46d
```

Comment:
In this flyteidl version is it possible to add the auth client initialisation similar to the AdminClient initialisation

https://github.com/flyteorg/flyteidl/blob/33c27f23cc33a97d688d078e750d836567b8d616/clients/go/admin/client.go#L147

We probably need a similar one inorder to call the auth API calls. 
I was able to call the new config calls for the auth in the flytectl MR but had to add the client initialization here
https://github.com/flyteorg/flytectl/pull/49/files#diff-da2ad8aa38e2333733df1c83cd6210e252c029f4425e70d8e9e7409fd645d312R18

We can probably move this to your flyteidl MR. Let me know what you think

Reply:
Have added the changes here https://github.com/flyteorg/flyteidl/pull/140 which requests a merge to your flyteidl branch. Please have a look